MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -525,17 +525,15 @@ internal class OfflineSearchEngineIntegrationTest {
                 createCoreSearchAddress(
                     houseNumber = "2011",
                     street = "15th Street Northwest",
```

Comment:
Offline Search Failing Tests:
Expected: "2011 15th Street Northwest, Washington, District of Columbia 20009"
It looks like there's an issue where we sometimes drop the "place" name or drop the region name.